notebooks/Vid2Desc_Gemini_1_0_Pro_Vision.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            raise ValueError("Either gcs or base64_encoded must be provided")
</pre><p> </p><p><code></code><code>python</p><p>raise ValueError("Either a GCS bucket path or base64_encoded string of the video must be provided")</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/Vid2Persona/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>